Gruntfile.js

```diff
@@ -72,7 +57,7 @@ module.exports = function(grunt) {
           sourceMap: false,
         },
         files: {
-          'build/lib/fxpay.debug.js': libFiles,
+          'build/lib/fxpay.debug.js': 'lib/**/*.js',
```

Comment:
This file building part will need to be replaced as change doesn't update the built file yet.